styles/lazy-styles.css

```diff
@@ -4,29 +4,29 @@
     src: local('☺'), url('fonts/HelveticaNeueLTStd-Bd.woff') format('woff'), url('fonts/HelveticaNeueLTStd-Bd.ttf') format('truetype');
     font-weight: normal;
     font-style: normal;
-    font-display: swap;
+    font-display: optional;
```

Comment:
Doesn't optional essentially let the browser opt-out of showing the custom font if it decides it's not ideal to show it (i.e. slow connection)? Won't we risk being stuck on the fallback font thus?

Reply:
true. I'll revert